app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.__class__.alive.append(self)
+
+    def __repr__(self):
+        return f"{'{'}Name: {self.name}, "\
+            f"Health: {self.health}, Hidden: {self.hidden}{'}'}"
```

Comment:
would be better to write like this:
```suggestion
        return f"{{Name: {self.name}," \
               f" Health: {self.health}," \
               f" Hidden: {self.hidden}}}"
```